src/test/java/seedu/guest/model/guest/DateRangeTest.java

```diff
@@ -42,4 +44,11 @@ public void isValidDateRange() {
         assertTrue(DateRange.isValidDateRange("30/09/22 - 01/10/22")); // across months
         assertTrue(DateRange.isValidDateRange("13/09/22 - 15/09/23")); // across years
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments.